### PR TITLE
Small tweaks and polish

### PR DESCRIPTION
**Pull request by thearst3rd**
* Fix some bugs from #49
* Enemies that die while attacking the castle still cause a wait
* New masks
* Fixed misplaced pixel
* Added sound to card drafting
* Polished the round-end a bit

### CODE REVIEW

File: assets/playfield.png
Comment (admoore0) [inline assets/playfield.png:1]: image

File: src/states/game/card_drafting.gd [hunk 1]
```diff
@@ -72,4 +74,7 @@ func select_card(selection: DualCardData) -> void:
 		draft_round = 1
 		Global.card_replay_moves = Global.card_current_moves
 		Global.card_current_moves = {}
+		$CenterContainer.hide()
+		modulate = Color(0.0, 0.0, 0.0, 0.0) # Look like we're hiding but prevent the mouse from passing through
```
Comment (admoore0) [inline src/states/game/card_drafting.gd]: Is there a constant for `Color.TRANSPARENT` or something?

**Review by admoore0 — CHANGES_REQUESTED** [review]

File: src/states/game/card_drafting.gd (hunk 1 above)
Comment (thearst3rd) [reply src/states/game/card_drafting.gd]: Good catch

**Review by admoore0 — DISMISSED** [review]

**Review by kdreese — DISMISSED** [review]

File: assets/playfield.png
Comment (thearst3rd) [reply assets/playfield.png:1]: 📠

**Review by kdreese — APPROVED** [review]

**Pull request merged**